docs/Chromaticqa-Guide.md

```diff
@@ -0,0 +1,20 @@
+# ChromaticQA
+
+## Introduction to ChromaticQA
+[Chromatic Docs](https://docs.chromaticqa.com/)
+
+ChromaticQA is a visual regression tool for storybook maintained by Storybook. The purpose of having chromaticQA run on all stories is to capture unwanted visual changes introduced by every change done to the code base. This reduces the overhead on manual regression as it supports screenshot comparisons for Chrome, IE and FF.
+
+## Running Test
+ChromaticQA runs as a part of the build. The builds can be seen here (https://www.chromaticqa.com/builds?appId=5d28eb3fe163f6002046d6fa)
+
+To run chromatic locally export the app code in bash profile(`export CHROMATIC_APP_CODE=<app-code>`) and run `npx chromatic test run  --build-script-name build:storybook || true`
```

Comment:
What is the use case for this? It is expensive to run these tests, I think running on every push should be more than enough.

Reply:
my use case for running it locally :
 
- if I'm developing something to fix a bug regarding our snapshots, then it is quicker to run chromatic tests locally than pushing my changes to a PR. (but I guess this command would only be used heavily in core-pod)

Reply:
I can only think of the above use case for running it locally

Reply:
Ok, I guess it's fine to leave it.